src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result -= 50;
```

Comment:
Replace the hard-coded value `50` with a constant that explains what this value represents (e.g., `LONG_TERM_DISCOUNT`). This enhances code readability and maintainability. See checklist item 2.